cmd/argo/commands/install.go

```diff
@@ -25,57 +25,62 @@ const clusterAdmin = "cluster-admin"
 
 func init() {
 	RootCmd.AddCommand(installCmd)
-	installCmd.Flags().StringVar(&installArgs.controllerName, "controller-name", common.DefaultControllerDeploymentName, "name of controller deployment")
-	installCmd.Flags().StringVar(&installArgs.uiName, "ui-name", common.DefaultUiDeploymentName, "name of ui deployment")
-	installCmd.Flags().StringVar(&installArgs.namespace, "install-namespace", common.DefaultControllerNamespace, "install into a specific namespace")
-	installCmd.Flags().StringVar(&installArgs.configMap, "configmap", common.DefaultConfigMapName(common.DefaultControllerDeploymentName), "install controller using preconfigured configmap")
-	installCmd.Flags().StringVar(&installArgs.controllerImage, "controller-image", common.DefaultControllerImage, "use a specified controller image")
-	installCmd.Flags().StringVar(&installArgs.uiImage, "ui-image", common.DefaultUiImage, "use a specified ui image")
-	installCmd.Flags().StringVar(&installArgs.executorImage, "executor-image", common.DefaultExecutorImage, "use a specified executor image")
-	installCmd.Flags().StringVar(&installArgs.serviceAccount, "service-account", "", "use a specified service account for the workflow-controller deployment")
+	installCmd.Flags().StringVar(&installArgs.ControllerName, "controller-name", common.DefaultControllerDeploymentName, "name of controller deployment")
+	installCmd.Flags().StringVar(&installArgs.UIName, "ui-name", common.DefaultUiDeploymentName, "name of ui deployment")
+	installCmd.Flags().StringVar(&installArgs.Namespace, "install-namespace", common.DefaultControllerNamespace, "install into a specific Namespace")
+	installCmd.Flags().StringVar(&installArgs.ConfigMap, "configmap", common.DefaultConfigMapName(common.DefaultControllerDeploymentName), "install controller using preconfigured configmap")
+	installCmd.Flags().StringVar(&installArgs.ControllerImage, "controller-image", common.DefaultControllerImage, "use a specified controller image")
+	installCmd.Flags().StringVar(&installArgs.UIImage, "ui-image", common.DefaultUiImage, "use a specified ui image")
+	installCmd.Flags().StringVar(&installArgs.ExecutorImage, "executor-image", common.DefaultExecutorImage, "use a specified executor image")
+	installCmd.Flags().StringVar(&installArgs.ServiceAccount, "service-account", "", "use a specified service account for the workflow-controller deployment")
 }
 
-type installFlags struct {
-	controllerName  string // --name
-	uiName          string // --ui-name
-	namespace       string // --install-namespace
-	configMap       string // --configmap
-	controllerImage string // --controller-image
-	uiImage         string // --ui-image
-	executorImage   string // --executor-image
-	serviceAccount  string // --service-account
+// InstallFlags has all the required parameters for installing Argo.
+type InstallFlags struct {
+	ControllerName  string // --name
```

Comment:
Can you fix this, comment. It actually refers to --controller-name?

Reply:
Done